### PR TITLE
Added combinato test

### PR DESCRIPTION
**Pull request by chyumin**

### CODE REVIEW

**Comment by bendichter** [comment]
why are you changing all the ss to si?

**Comment by chyumin** [comment]
> why are you changing all the ss to si?

@bendichter Nothing specific, just using new code api/syntax. Any caveats on that?

File: tests/test_singularity_containers.py [hunk 1]
```diff
@@ -38,40 +37,48 @@ def run_kwargs(work_dir):
...
 def test_kilosort1(run_kwargs):
-    sorting = ss.run_kilosort(output_folder="kilosort", useGPU=False, **run_kwargs)
+    sorting = si.run_kilosort(output_folder="kilosort", useGPU=False, **run_kwargs)
+    print(sorting)
+
+
+def test_combinato(run_kwargs):
+    rec = run_kwargs['recording']
+    channels = rec.get_channel_ids()[0:1]
+    run_kwargs['recording'] = si.ChannelSliceRecording(rec, channels)
```
Comment (alejoe91) [inline tests/test_singularity_containers.py:82]:
```suggestion
    rec_one_channel = rec.channel_slice(channels)
    run_kwargs['recording'] = rec_one_channel
```

**Comment by alejoe91** [comment]
> > why are you changing all the ss to si?
> 
> @bendichter Nothing specific, just using new code api/syntax. Any caveats on that?

@chyumin I think we can keep the `se`/`ss` notation. You can check my comment on how to slice without using `si.ChannelSliceRecording`, but let's wait for tests to finish first

**Pull request merged**